editor/editor_inspector.h

```diff
@@ -313,6 +313,8 @@ class EditorInspector : public ScrollContainer {
 	String property_prefix; //used for sectioned inspector
 	String object_class;
 
+	bool restrict_to_basic = false;
```

Comment:
This should be `true` (explains why @KoBeWi saw that advanced settings are shown initially even though the default mode is basic).

Reply:
No, it's ok that inspector itself does not restrict to basic. Also this is being set in the constructor, so it might be something else.